hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceConfigurationBuilder.java

```diff
@@ -971,13 +973,16 @@ private static void determineResourceTypeSetDmr(ModelNode config,
                             List<Property> operationList = opModelNode.asPropertyList();
                             for (Property operationProperty : operationList) {
                                 ModelNode operationValueNode = operationProperty.getValue();
-                                String operationName = operationProperty.getName();
+                                String name = operationProperty.getName();
+
+                                List<OperationParam> params = getOpParamListFromOpNode(operationValueNode);
 
                                 PathAddress pathAddress = getPath(operationValueNode, context,
                                         DMROperationAttributes.PATH);
-                                Operation<DMRNodeLocation> op = new Operation<>(ID.NULL_ID, new Name(operationName),
+                                String operationName = getString(operationValueNode, context, DMROperationAttributes.OPERATION_NAME);
+                                Operation<DMRNodeLocation> op = new Operation<>(ID.NULL_ID, new Name(name),
```

Comment:
this changes the operation ID from the internal name to the display name. I think that is incorrect (?) When NULL is passed in the ID param, the Name also becomes the ID. To keep the ID the same as it was before, you'd need to change the first argument from ID.NULL_ID to "new ID(operationName)"